langchain-core/src/runnables/base.ts

```diff
@@ -539,7 +539,7 @@ export abstract class Runnable<
    */
   pipe<NewRunOutput>(
     coerceable: RunnableLike<RunOutput, NewRunOutput>
-  ): RunnableSequence<RunInput, Exclude<NewRunOutput, Error>> {
+  ): Runnable<RunInput, Exclude<NewRunOutput, Error>> {
     // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
I believe this should be `RunnableInterface<RunInput, Exclude<NewRunOutput, Error>>`. cc @jacoblee93 

Same with the below two

Reply:
Never mind, this is not the case